dept44-starter/src/main/java/se/sundsvall/dept44/util/MunicipalityUtils.java

```diff
@@ -0,0 +1,90 @@
+package se.sundsvall.dept44.util;
+
+import static java.util.Comparator.comparing;
+import static org.apache.commons.lang3.StringUtils.upperCase;
+import static org.springframework.util.ResourceUtils.getURL;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class MunicipalityUtils {
```

Comment:
För formens skull: borde vara `final` va?

Reply:
Of course! Fixed!